server/routerlicious/packages/services-shared/src/http.ts

```diff
@@ -67,7 +67,13 @@ export function handleResponse<T>(
 				response.setHeader("Cache-Control", "no-store, max-age=0");
 			}
 
+			response.setHeader(
```

Comment:
It's probably fine to have 3 different place to have same code, but maybe there is a way to expose shared function that adds performance-related headers to any response?

Reply:
I talked to Zach exactly about this and, as he is planning on removing the duplicated code from historian in the near futura, .  he let me know it is ok to leave it as it is. We would have more work than it is worth it to expose this new function to historian when eventually the duplicate code will go away anyway.